config/accounts.go

```diff
@@ -1,9 +1,61 @@
 package config
 
+// IntegrationType : Integration type enumeration
+type IntegrationType string
+
+// The integration types
```

Comment:
Are we going to use a linter? If so, golint says that this section should have a comment because the constants are exported.

Reply:
Interesting, normally the linting rules say that the comment should start with the variable/function name. In this case should each lntegrationType have a comment? In any case, the comment above really doesn't say much beyond "This is here to make the linter happy" and it seems doubtful it would make a linter happy as written.